backend/src/main/java/com/relay/config/WebSecurityConfig.java

```diff
@@ -25,36 +22,35 @@ public UserDetailsService userDetailsService() {
                 .roles("USER", "ADMIN").build());
         return manager;
     }
-    //
-    // @Autowired
-    // public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
-    //
-    // auth.inMemoryAuthentication().withUser("user") // #1
-    // .password("{noop}password").roles("USER").and().withUser("admin") // #2
-    // .password("{noop}password").roles("ADMIN", "USER");
-    // }
 
-    @Bean
-    public ReactiveUserDetailsService userDetailsRepository() {
-
-        UserDetails user =
-                User.withUsername("user").password("{noop}password").roles("USER").build();
-        return new MapReactiveUserDetailsService(user);
-    }
+    @Autowired
+    public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
 
-    @Bean
-    protected SecurityWebFilterChain configure(ServerHttpSecurity http) {
-
-        http.httpBasic().disable();
-        http.formLogin().disable();
-        http.csrf().disable();
-        http.logout().disable();
-        // http.authorizeExchange().anyExchange().permitAll()
-        // // .and().formLogin().loginPage("/login")
-        // .and().csrf().disable();
-        // return http.build();
-        return http.authorizeExchange().anyExchange().permitAll().and().build();
+        auth.inMemoryAuthentication().withUser("user") // #1
+                .password("{noop}password").roles("USER").and().withUser("admin") // #2
+                .password("{noop}password").roles("ADMIN", "USER");
     }
 
+    // @Bean
+    // public ReactiveUserDetailsService userDetailsRepository() {
+    //
+    // UserDetails user =
+    // User.withUsername("user").password("{noop}password").roles("USER").build();
+    // return new MapReactiveUserDetailsService(user);
+    // }
+    //
+    // @Bean
+    // protected SecurityWebFilterChain configure(ServerHttpSecurity http) {
+    //
+    // http.httpBasic().disable();
+    // http.formLogin().disable();
+    // http.csrf().disable();
+    // http.logout().disable();
+    // // http.authorizeExchange().anyExchange().permitAll()
+    // // // .and().formLogin().loginPage("/login")
+    // // .and().csrf().disable();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)